sw/device/lib/crypto/drivers/BUILD

```diff
@@ -24,7 +24,7 @@ cc_library(
     hdrs = ["aes.h"],
     deps = [
         ":entropy",
-        "//hw/ip/aes/data:aes_c_regs",
+        "//hw/top:aes_c_regs",
```

Comment:
Too bad the dep below is top-specific. I think it could be expressed in a top-independent way?

Reply:
I am not entirely sure I understand your comment @matutem . Are you saying that you would prefer that `aes_c_regs` was not a top-specific dependency?